envs/transformers-env.yaml

```diff
@@ -2,16 +2,21 @@ builder_version: ">=10.0.1"
 
 packages:
 {% if not s390x %}
-  - feedstock : https://github.com/conda-forge/rust-feedstock
-    git_tag : 97bf9eebc206d17594013dd0e89938f6536a783a     # 1.60
-    patches :
-      - feedstock-patches/rust/0001-Fixed-build.patch
-  - feedstock : https://github.com/conda-forge/rust-activation-feedstock
-    git_tag : 75803486a2dbca1b39b2a0594b682afaabcb5b13
-    patches :
-      - feedstock-patches/rust-activation/0001-Fixed-build.patch
-  - feedstock : https://github.com/conda-forge/setuptools-rust-feedstock
-    git_tag : 99db82b53099cbc1315238c835f3ad6bc7413774
+  - feedstock : https://github.com/conda-forge/rust-feedstock     # [build_type == 'cpu' or cudatoolkit == '11.4']
+    git_tag : 97bf9eebc206d17594013dd0e89938f6536a783a            # [build_type == 'cpu' or cudatoolkit == '11.4']
+    patches :                                                     # [build_type == 'cpu' or cudatoolkit == '11.4'] 
+      - feedstock-patches/rust/0001-Fixed-build.patch             # [build_type == 'cpu' or cudatoolkit == '11.4']
+  - feedstock : https://github.com/conda-forge/rust-activation-feedstock # [build_type == 'cpu' or cudatoolkit == '11.4']
+    git_tag : 75803486a2dbca1b39b2a0594b682afaabcb5b13            # [build_type == 'cpu' or cudatoolkit == '11.4']
+    patches :                                                     # [build_type == 'cpu' or cudatoolkit == '11.4']
+      - feedstock-patches/rust-activation/0001-Fixed-build.patch  # [build_type == 'cpu' or cudatoolkit == '11.4']
+  - feedstock : https://github.com/conda-forge/setuptools-rust-feedstock # [build_type == 'cpu' or cudatoolkit == '11.4']
+    git_tag : 99db82b53099cbc1315238c835f3ad6bc7413774            # [build_type == 'cpu' or cudatoolkit == '11.4']
+
+  - feedstock : https://github.com/conda-forge/rust-feedstock.git # [cudatoolkit == '11.2']
```

Comment:
I think lets just mention version of rust in comment as we have it twice in this env.